src/Types.hs

```diff
@@ -1071,6 +1084,8 @@ countIdeaVotes v = countEq v ideaVoteValue
 countCommentVotes :: UpDown -> CommentVotes -> Int
 countCommentVotes v = countEq v commentVoteValue
 
+-- Given a list of parents and a collection (AMap) of comments
+-- returns the collection of comments after following the parenting list.
 traverseParents :: [AUID Comment] -> Traversal' Comments Comments
```

Comment:
I am skeptical of plural type synonyms such as `Comments`, but since it hasn't been introduced here, we don't have to remove it here.  Just saying.